panels/export.py

```diff
@@ -0,0 +1,69 @@
+from bpy.props import *
+from bpy.types import (Panel, PropertyGroup)
+
+from ..operators import (ExportUploadOperator, OAuthOperator)
+from ..globals import APP_CATEGORY
+
+SOURCE_TYPES = [
+    ("3D_MODEL", "3D Model", "Models designed through blender", "FILE_3D", 1),
+    ("3D_CAPTURE", "3D Capture",
+     "Model created through photogrammetry processes", "CAMERA_DATA", 2)
+]
+
+
+class ExportProperties(PropertyGroup):
+    name: StringProperty(name="Name", description="Title for the upload")
+    description: StringProperty(
+        name="Description",
+        description="Give an overview of the contents of the data")
+    attribution: StringProperty(
+        name="Attribution",
+        description="Acknowledge the particular author, artist, or coporation")
+    source_type: EnumProperty(name="Model Type",
+                              description="Type of data being uploaded",
+                              items=lambda self, context: SOURCE_TYPES)
+    webp_textures: BoolProperty(
+        name="Convert to WebP",
```

Comment:
I think this is confusing since it doesn't say what will be WebP.  `Use WebP images` is what the ion UX has, so let's go with that.